src/posts/unlock-your-continuous-delivery-potential-vsm.md

```diff
@@ -0,0 +1,125 @@
+
+The interactive nature of Value Stream Mapping is great for getting those responsible for activities involved in discussing waste and possible solutions right at the beginning. This allows people to take ownership of their work and be involved in shaping the future process.
+
+### Lack of feedback
+
+Value Stream Maps do not provide feedback about changes you implement directly but do provide guidance for what to measure. Asking the right question can be very difficult and Value Stream Maps make asking questions such as “did change X help solve waste Y?” a lot easier to write, communicate and measure.
+
+Remember that metrics are only part of the feedback. The Developer Experience is not easy to measure with metrics (although they can be correlated, e.g. better performance typically leads to happier teams and vice versa). By referring back to the Value Stream Map and the waste that was identified, it makes it easier to discuss feedback from the team.
+
+It is important to decompose problems into small incremental changes. This is much easier to do when the waste we have identified is related to steps in the release process. For example, sometimes waste is not a problem but a symptom of other waste. We can consider this waste to be transitive waste. This is valuable information when making changes because it can help us to order and plan changes incrementally.
+
+### Lack of support from management
+
+Waste identified in the Value Stream Map can represent the cost of inefficiencies in your current release process. Quite simply; if you remove this waste then that is your return on investment. This makes it much easier to justify investment into improvements.
+
+It is recommended to include management in the Value Stream Mapping process; it’s a great way for them to learn more about how the teams work together. Once you start to tackle the waste it’s also much easier to plan the work into the ongoing project schedules. This is great because it means you can be quite flexible with how much time you invest into improvements based on how much you think it will benefit you.
+
+### Following best practices
+
+If you use Value Stream Mapping to help guide your Continuous Delivery journey, you will better understand what your team’s individual challenges are and better reason about possible solutions. Once you know ***why*** you are doing something, it’s much easier to start looking for best practices that try to solve the same thing. Love the problem, not the solution. There is no endgame with Continuous Delivery and time is finite.
+
+> Love the problem, not the solution. There is no endgame with Continuous Delivery and time is finite.
```

Comment:
Whilst not native Markdown, this is how we have formatted quotes in the past:
https://github.com/verifa/website/blob/2bcdf26dfb89f42c035e5f202c9adc38acfa18ce/src/posts/cases/value-stream-mapping-with-visma.md?plain=1#L45

```html
<blockquote>
  ‘The Value Stream clearly visualises our delivery process (and its pains). Verifa helped lead discussions around many topics which was useful in gaining a common understanding of what we can do to improve.’
  <br/>
  <cite>- Olli Suihkonen, Senior Manager, Visma Software Oy.</cite>
</blockquote>
```

We should use this in our blogs because the formatting is pretty :)

Reply:
I like pretty formatting :D